tests/files/api_fixtures/gcp_set_subscription_admin_verified.yaml

```diff
@@ -0,0 +1,236 @@
+        "gcp_connected": true,
+        "gcp_paused": false,
+        "gcs_manager_url": null,
+        "gcs_version": null,
+        "globus_connect_setup_key": null,
+        "guest_collection_activity_notification_policy": null,
+        "high_assurance": false,
+        "host_endpoint": "u_xhavtnb4b5g3jkh632ewsb7eeu#7a2b1f42-d232-4b64-b071-7f5efa0f06d5",
+        "host_endpoint_display_name": "GCP Standard Mapped Collection on MacOS",
+        "host_endpoint_id": "7a2b1f42-d232-4b64-b071-7f5efa0f06d5",
+        "host_path": "/Users/nobody/Public/",
+        "https_server": null,
+        "id": "33d7a15d-8776-43a5-9e2d-cf25b92cd2a6",
+        "in_use": false,
+        "info_link": null,
+        "is_globus_connect": false,
+        "keywords": null,
+        "last_accessed_time": null,
+        "local_user_info_available": null,
+        "location": null,
+        "mapped_collection_display_name": null,
+        "mapped_collection_id": null,
+        "max_concurrency": null,
+        "max_parallelism": null,
+        "mfa_required": false,
+        "my_effective_roles": [
+          "administrator",
+          "access_manager",
+          "activity_manager",
+          "activity_monitor"
+        ],
+        "myproxy_dn": null,
+        "myproxy_server": "myproxy.globusonline.org",
+        "name": "33d7a15d-8776-43a5-9e2d-cf25b92cd2a6",
+        "network_use": null,
+        "non_functional": false,
+        "non_functional_endpoint_display_name": null,
+        "non_functional_endpoint_id": null,
+        "oauth_server": null,
+        "organization": null,
+        "owner_id": "9ef25e9f-32eb-411f-812a-4b090b9f1908",
+        "owner_string": "nobody@globus.org",
+        "preferred_concurrency": null,
+        "preferred_parallelism": null,
+        "preserve_modtime": null,
+        "public": true,
+        "requester_pays": false,
+        "restrict_transfers_to_high_assurance": null,
+        "s3_owner_activated": false,
+        "s3_url": null,
+        "shareable": false,
+        "sharing_target_endpoint": "u_xhavtnb4b5g3jkh632ewsb7eeu#7a2b1f42-d232-4b64-b071-7f5efa0f06d5",
+        "sharing_target_root_path": "/Users/nobody/Public/",
+        "skip_auto_delete": null,
+        "subscription_admin_verified": false,
+        "subscription_id": "aa4d684f-29d0-4b07-8a03-4f8cd570f982",
+        "tlsftp_server": null,
+        "user_message": null,
+        "user_message_link": null,
+        "username": "u_xhavtnb4b5g3jkh632ewsb7eeu"
+      }
+  - path: /endpoint/7a2b1f42-d232-4b64-b071-7f5efa0f06d5/subscription_admin_verified
+    method: put
+    json:
+      {
+        "DATA_TYPE": "result",
+        "code": "Updated",
+        "message": "Endpoint updated successfully",
+        "request_id": "9jFjlPyEQ",
+        "resource": "/endpoint/7a2b1f42-d232-4b64-b071-7f5efa0f06d5/subscription_admin_verified",
+      }
+  - path: /endpoint/33d7a15d-8776-43a5-9e2d-cf25b92cd2a6/subscription_admin_verified
+    method: put
+    json:
```

Comment:
My guess is that this would also need a non-successful HTTP status, probably 403, right?

I think that's going to change your testing of this, but I believe that the generic handler for Transfer API errors will produce good output in this case.

Reply:
This appears to be resolved.